src/main/java/uk/gov/hmcts/reform/wataskmanagementapi/services/TaskManagementService.java

```diff
@@ -637,6 +637,14 @@ public void terminateTask(String taskId, TerminateInfo terminateInfo) {
                 //Commit transaction
                 cftTaskDatabaseService.saveTask(task);
                 break;
+            case DELETED:
+                //Update cft task
+                task.setState(CFTTaskState.TERMINATED);
+                //Perform Camunda updates
+                camundaService.deleteCftTaskState(taskId);
+                //Commit transaction
+                cftTaskDatabaseService.saveTask(task);
+                break;
```

Comment:
Maybe we need a fallback rule?  e.g. If it's not a known reason maybe that should be set to Terminated.

Reply:
We may also want to check the states we are setting: https://tools.hmcts.net/confluence/pages/viewpage.action?pageId=1518684941#HLDTaskRepositoryv1.1-2.5.2CFTTaskState

Reply:
This could be a clean solution to default to terminated, happy to implement if needed

Reply:
Discussion was taken offline and a new ticket was raised to attempt to resolve and cater for future error scenarios, 
Thanks @raghera